journal/202104.md

```diff
@@ -0,0 +1,351 @@
+# {DRAFT} Decred Journal – April 2021
+
+![{alt text}](../img/{file.ext} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+April's highlights: {
+
+- pi v1.0 and proposal
+- Decrediton DEX integration progress
+- consensus vote/upgrade status
+
+}
+
+Contents: { sync }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Integrations](#integrations)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Community Discussions](#community-discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+## Upgrade reminder
+
+{}
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+- codebase [converted](https://github.com/decred/dcrd/pull/2625) to use the new `stdaddr` package for handling addresses. As common for large changes, to make review easier this one was split into a series of individual commits such that everything builds and passes all tests each step of the way. { "make use of the script version" - how is this different from not making use of it previously? }
```

Comment:
Previous to this change, all code that dealt with addresses assumed they were version 0 because that is the only version that is currently supported.  The relevant code now supports other versions by using the version that is now explicitly returned as part of the new address decoding logic instead of assuming it's 0.  In other words, it's one step closer to being able to cleanly introduce new scripting language versions that are not limited to relatively trivial changes.

Reply:
"new scripting language versions" is a must-mention, thanks!